File: 37-day/randomNumbers.js
function randomNumbers2(dices, faces) {
    let results = new Array()
    for (let d = 0; d < dices; d++) {
        results.push(Math.ceil(Math.random() * faces))
    }
    return results
}

const numberDices = parseInt(prompt('How many dice are you going to roll? '))
const numberFaces = parseInt(prompt('How many faces do they have? '))
alert(`Point: ${randomNumbers2(numberDices, numberFaces)}`)

File: 37-day/randomNumbers.py
form random import choice

def randomNumbers2(dices, faces):
    results = []
    for d in range(0, dices):
        results.append(choice(range(1, faces + 1)))
    return results

numberDices = init(input('How many dice are you going to roll? '))
numberFaces = init(input('How many faces do they have? '))
print('Point: {}'.format(randomNumbers2(numberDices, numberFaces)))